applications/RANSApplication/custom_utilities/rans_variable_utilities.cpp

```diff
@@ -241,6 +176,159 @@ void CopyNodalSolutionStepVariablesList(
 
     KRATOS_CATCH("");
 }
+
+template <typename TDataType>
+void AssignConditionVariableValuesToNodes(
+    ModelPart& rModelPart,
+    const Variable<TDataType>& rVariable,
+    const Flags& rFlag,
+    const bool FlagValue)
+{
+    auto& r_nodes = rModelPart.Nodes();
+    VariableUtils().SetHistoricalVariableToZero(rVariable, r_nodes);
+
+    block_for_each(rModelPart.Conditions(), [&](ModelPart::ConditionType& rCondition) {
+        if (rCondition.Is(rFlag) == FlagValue) {
+            const int number_of_nodes = rCondition.GetGeometry().PointsNumber();
+            const auto& r_normal = rCondition.GetValue(rVariable);
+            for (int i_node = 0; i_node < number_of_nodes; ++i_node) {
+                auto& r_node = rCondition.GetGeometry()[i_node];
+                r_node.SetLock();
+                r_node.FastGetSolutionStepValue(rVariable) +=
+                    r_normal * (1.0 / static_cast<double>(number_of_nodes));
+                r_node.UnSetLock();
+            }
+        }
+    });
+
+    rModelPart.GetCommunicator().AssembleCurrentData(rVariable);
+}
+
+void AddAnalysisStep(
+    ModelPart& rModelPart,
+    const std::string& rStepName)
+{
+    auto& r_process_info = rModelPart.GetProcessInfo();
+    if (!r_process_info.Has(ANALYSIS_STEPS))
+    {
+        r_process_info.SetValue(ANALYSIS_STEPS, std::vector<std::string>());
+    }
+    r_process_info[ANALYSIS_STEPS].push_back(rStepName);
+}
+
+bool IsAnalysisStepCompleted(
+    const ModelPart& rModelPart,
+    const std::string& rStepName)
+{
+    const auto& r_process_info = rModelPart.GetProcessInfo();
+    if (r_process_info.Has(ANALYSIS_STEPS))
+    {
+        const std::vector<std::string>& r_steps = r_process_info[ANALYSIS_STEPS];
+        return (std::find(r_steps.begin(), r_steps.end(), rStepName) != r_steps.end());
+    }
+    else
+    {
+        return false;
+    }
+}
+
+void AssignBoundaryFlagsToGeometries(
+    ModelPart& rModelPart)
+{
+    block_for_each(rModelPart.Conditions(), [&](ModelPart::ConditionType& rCondition) {
+        rCondition.SetValue(RANS_IS_INLET, rCondition.Is(INLET));
+        rCondition.SetValue(RANS_IS_OUTLET, rCondition.Is(OUTLET));
+        rCondition.SetValue(RANS_IS_STRUCTURE, rCondition.Is(STRUCTURE));
+    });
+}
+
+void CalculateMagnitudeSquareForNodal3DVariable(
+    ModelPart& rModelPart,
+    const Variable<array_1d<double, 3>>& r3DVariable,
+    const Variable<double>& rOutputVariable)
+{
+    block_for_each(rModelPart.Nodes(), [&](ModelPart::NodeType& rNode) {
+        const double magnitude = norm_2(rNode.FastGetSolutionStepValue(r3DVariable));
+        rNode.FastGetSolutionStepValue(rOutputVariable) = std::pow(magnitude, 2);
```

Comment:
```suggestion
        const auto& r_val = rNode.FastGetSolutionStepValue(r3DVariable);
        rNode.FastGetSolutionStepValue(rOutputVariable) = r_val[0] * r_val[0]  + r_val[1] * r_val[1] + r_val[2] * r_val[2];
```

Reply:
removed the method. It is no longer used.